docs/1.0/demos.md

```diff
@@ -0,0 +1,20 @@
+# Demos
```

Comment:
@timmixell what was the reasoning for moving the demos under docs?  I may not be typical, but when I'm downloading libraries and want to clean things up (free up space, etc.), the first thing I get rid of are demos once I don't need them anymore.  With this structure, I'd have to go into each version under docs, rather than just having a simple "demos" folder.

Reply:
@dsbrady , i guess we weren't really thinking of the API docs as something someone would actually download, so much as use a reference material or request new features. 

what structure would you propose?

Reply:
You may be right. I was thinking of this more along the lines of our other GH repos which have other code.  I retract my question.